webapp/channels/src/images/browser-icons/firefox.svg

```diff
@@ -68,7 +68,7 @@
       <stop offset="0" stop-color="#c42482" stop-opacity="0.5"/>
       <stop offset="0.474" stop-color="#ff272d" stop-opacity="0.5"/>
       <stop offset="0.486" stop-color="#ff2c2c" stop-opacity="0.513"/>
-      <stop offset="0.675" stop-color="#ff7a1a" stop-opacity="0.72"/>
+      <stop offset="0.675" stop-color="#ff7a1a" stop-opacity="0.73"/>
```

Comment:
Did you mean to change this value? This seems like it might've gotten accidentally hit by a find and replace

Reply:
Unintented, good catch!